### PR TITLE
Fix team names for cirun

### PR DESCRIPTION
**Pull request by isuruf**
Checklist
* [x] Added a ``news`` entry

### CODE REVIEW

**Review by beckermr — APPROVED** [review]

File: news/cirun.rst [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+**Removed:**
+
+* <news item>
+
+**Fixed:**
+
+* The team name for cirun was fixed. Previously the team name passed had
+  -feedstock in it and also did not support teams as maintainers.
+  For teams like conda-forge/r, if they are added to a feedstock after
+  Cirun is configured, the feedstock needs to be reconfigured.
```
Comment (isuruf) [inline news/cirun.rst:22]:
```suggestion
  Cirun is configured, the feedstock needs to be reconfigured (#1794).
```

**Review by jaimergp — APPROVED** [review]
Thanks!

**Comment by isuruf** [comment]
I've removed the unrelated changes. Will send a new PR.

**Comment by isuruf** [comment]
Thanks for the reviews

**Pull request merged**